Sources/ToastView.swift

```diff
@@ -156,6 +160,9 @@ open class ToastView: UIView {
       height = containerSize.width
       y = self.bottomOffsetLandscape
     }
+    if #available(iOS 11.0, *), useSafeAreaForBottomOffset {
+      y += ToastWindow.shared.safeAreaInsets.bottom
```

Comment:
We should make sure that it will work in both landscape and portrait... and maybe even portrait upside-down...

Reply:
Okay, I will check it again 🙂

Reply:
@Coeur I have checked that Toast is shown properly in all orientation.

```swift
let appearance = ToastView.appearance()
appearance.useSafeAreaForBottomOffset = true
appearance.bottomOffsetPortrait = 0
appearance.bottomOffsetLandscape = 0
```

![ToasterWithSafeArea-iPhone8](https://user-images.githubusercontent.com/11647461/62845448-081a3e80-bd03-11e9-8701-1c265b55b9e6.gif)
iPhone8

![ToasterWithSafeArea-iPhoneX](https://user-images.githubusercontent.com/11647461/62845443-fdf84000-bd02-11e9-844c-6b116f8af6d5.gif)
iPhoneX

![ToasterWithSafeArea-iPad](https://user-images.githubusercontent.com/11647461/62845502-84148680-bd03-11e9-8e7c-0bc076add2f4.gif)
iPad Pro

Reply:
In your three animations, the only one with a notch is the iPhone X, and you're showing the scenario where upside-down is disabled... ;)

Reply:
@Coeur I tried to rotate upside-down in iPhoneX, but I realize iPhoneX device doesn't support upside-down 😢.
https://stackoverflow.com/a/46859396
https://forums.developer.apple.com/message/268015

Reply:
ah, oh, lol ... maybe because for this kind of scenario, ah ah!